website/templates/nav.mako

```diff
@@ -24,7 +24,7 @@
     <div id="navbar" class="navbar-collapse collapse navbar-right">
       <ul class="nav navbar-nav">
         % if user_name:
-            <li id="osfNavDashboard"><a href="/">Dashboard</a></li>
+            <li id="osfNavDashboard"><a href="/dashboard">Dashboard</a></li>
```

Comment:
We want every `/dashboard` to go through the new view `dashboard` which has a @must_be_logged_in` decorator before the redirection. This can trigger consistent CAS redirection (auto or manual depending on the status OSF cookie `osf` and CAS cookie `TGC`).